### DISCUSSION TITLE
Can i make VM's started fromt he command line visible in UTM?

### DISCUSSION THREAD

**Discussion by ToontjeM** [utmapp/UTM#6377] [UTM Help (macOS)]
I want to automate VM provisioning using Vagrant or Terraform. I can spin up Qemu VM's, but i don't see them in UTM. I want to use the UTM UI in the same way as VirtualBox uses it's UI. Whenever i spin up a VirtualBox VM i can see this VM in the ViertualBox UI.
Can i do the same woth Qemu VM's? Can i provision them from the command line and make them show up in UTM?

**Comment by naveenrajm7** [comment]
Hello, 

UTM comes with its own [fork of qemu](https://github.com/utmapp/qemu). So if you use the qemu, that you installed to spin up the VM. This VM will not be managed by UTM, and hence you will not see it in UTM UI.

If you are looking for vagrant + UTM , check out the plugin https://naveenrajm7.github.io/vagrant_utm/ which manages VMs using UTM.
